changelog.txt

```diff
@@ -8,6 +8,7 @@
 * Add - Tracking for returning from OAuth connection.
 * Fix - Transactions and deposits counts on the table summary are rendered as "undefined".
 * Update - Deposit overview details.
+* Add - Refresh cached account properties when Stripe external accounts are changing.
```

Comment:
Let's not mention Stripe here.
```suggestion
* Add - Refresh cached account on deposit bank accounts changes.
```

Reply:
Sound good, updated in 8d3d6250ad9a8c6654bc9cd842ef92af1e3f9153